### PR TITLE
SW-4377 Search table for facility inventory totals

### PR DESCRIPTION
**Pull request by sgrimm**
We're going to add a view to the web app that shows the total number of seedlings
across all species at each nursery. Currently, the inventory search tables all
group the totals by species ID. Add a new search table that ignores species ID and
aggregates all the batches at a facility. It includes a count of the total number
of species.

The new table can be searched using a payload like

    {
      "prefix": "facilities.facilityInventoryTotals",
      "fields": [
        "facility_id",
        "facility_name",
        "germinatingQuantity",
        "notReadyQuantity",
        "readyQuantity",
        "totalQuantity",
        "totalSpecies"
      ],
      "search": {
        "operation": "field",
        "field": "organization_id",
        "values": [
          "123"
        ]
      }
    }

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1447** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1447" target="_blank">Graphite</a> 
    * **PR #1448** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1448" target="_blank">Graphite</a> 
      * **PR #1449** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1449" target="_blank">Graphite</a> 
        * **PR #1450** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1450" target="_blank">Graphite</a>  👈
          * **PR #1453** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1453" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1450?utm_source=stack-comment).

File: src/main/resources/i18n/Messages_en.properties [hunk 1]
```diff
@@ -306,6 +306,11 @@ search.facilityInventories.germinatingQuantity=Germinating quantity (facility in
 search.facilityInventories.notReadyQuantity=Not Ready quantity (facility inventory)
 search.facilityInventories.readyQuantity=Ready quantity (facility inventory)
 search.facilityInventories.totalQuantity=Total quantity (facility inventory)
+search.facilityInventoryTotals.germinatingQuantity=Germinating quantity (total facility inventory)
```
Comment (karthikbtf) [inline src/main/resources/i18n/Messages_en.properties:309]: I can't recall how/where the search.table.field strings get used.

**Review by karthikbtf — APPROVED** [review]
Looks clear to me 👍

File: src/main/resources/i18n/Messages_en.properties (hunk 1 above)
Comment (sgrimm) [reply src/main/resources/i18n/Messages_en.properties:309]: > I can't recall how/where the search.table.field strings get used.

They are the column headers in CSV exports.

**Comment by sgrimm** [comment]
### Merge activity

* **Nov 2, 3:33 PM**: @sgrimm started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1450).
* **Nov 2, 3:35 PM**: [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1450) rebased this pull request as part of a merge.
* **Nov 2, 3:36 PM**: @sgrimm merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1450).

**Pull request merged**